### ISSUE TITLE
ImportError: No Module named nltk

### ISSUE THREAD

**Issue by BenLuker** [renpy/renpy#1924]
I have a Python file I have created within my game folder. I want to use NLTK (https://www.nltk.org/) in my Python file.

I am able to import that python file into Ren'Py just fine to use in my game _without_ NLTK like this:

`
init python:
    from myScript import *
`

I then installed NLTK using the instructions listed here: https://www.nltk.org/install.html
I have also installed the NLTK data using the instructions listed here: https://www.nltk.org/data.html

In my Python file I have this:

`
from nltk.corpus import brown
`

When I run my Python script from the command line everything works as intended. But when I try to launch my Ren'Py project I am presented with this error:

`
ImportError: No Module named nltk.corpus
`

I have tried a couple of things to get this working. Firstly, I tried moving my nltk_data folder straight into the games directory. I get the same error. Secondly, I tried importing it directly from the Ren'Py file in addition to the python file. I am again presented with the same error.

So now I'm wondering if I'm missing something big considering I can't find _anyone else_ with this problem. Apologies if this is a silly question but I genuinely am having a hard time finding others with the same problem and am struggling to get this to work.

**Comment by renpytom** [comment]
In general, you can't just import arbitrary python modules into Ren'Py and expect them to work, and so I can't provide support for Python modules that aren't distributed as part of Ren'Py. 

That being said, some pure-python modules can be installed using the instructions at:

https://www.renpy.org/doc/html/python.html#first-and-third-party-python-modules-and-packages

**Comment by renpytom** [comment]
https://github.com/renpy/renpy/commit/13a755ae934c33b9bbc9d888ffaf146178b104e3 has a fix that gets NLTK imported.

**Comment by BenLuker** [comment]
> In general, you can't just import arbitrary python modules into Ren'Py and expect them to work, and so I can't provide support for Python modules that aren't distributed as part of Ren'Py.
> 
> That being said, some pure-python modules can be installed using the instructions at:
> 
> https://www.renpy.org/doc/html/python.html#first-and-third-party-python-modules-and-packages

Ah right. I didn't quite understand what this was saying at first but now I get it, thank you! Okay so I followed the instructions listed there but then I got more import errors. This time, however, it was with a few .pyo files. So I downloaded those files and placed them into my renpy\lib\windows-i686\Lib folder and those errors seem to have gone away. (I hope I did that correctly lol)

Now that I did that, I'm getting an error saying
`SyntaxError: (unicode error) 'utf8' codec can't decode byte 0xe7 in position 0: unexpected end of data (python-packages/nltk/metrics/aline.py, line 81)`

Now, this seems to be what you addressed here.

> [13a755a](https://github.com/renpy/renpy/commit/13a755ae934c33b9bbc9d888ffaf146178b104e3) has a fix that gets NLTK imported.

So guessing from the name of the commit "IF a pure-python module can't be parsed as utf-8, try it as latin-1", this newest nightly build should fix this error specifically? 

I've downloaded the newest nightly build (assuming I did it correctly) and I'm still getting this error so I just want to make sure I'm on the right track with my thinking and I'm not missing something obvious like I was previously.
